recipe/build.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+# remove `-std=c++17` from CXXFLAGS
+export CXXFLAGS=$(echo $CXXFLAGS | sed -E 's@\-std=[^ ]+@@')
```

Comment:
Is this what you are trying to do here?
`export CXXFLAGS=${CXXFLAGS/-std=c++17/} # remove`
or
`export CXXFLAGS=${CXXFLAGS/-std=c++17/-std=c++14} # rename`

Reply:
Cool, I didn't know about that! I want to make sure it gets removed and is insensitive to the right hand side, so if it gets updated, this doesn't change. Can the `${foo//}` replacement take expressions or wildcards? If not, then this code is doing what it means to do.

Reply:
> Can the `${foo//}` replacement take expressions or wildcards?

Yeah, I think so.